client/src/components/SignUpPage.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div>
+    <Navbar />
+    <div class="container">
+      <div class="signup-form-container">
+        <form>
+          <input class="input-textarea" type="text" placeholder="E-mail" />
+          <input class="input-textarea" type="text" placeholder="Passsword" />
+          <input class="input-textarea" type="text" placeholder="Confirm password" />
+          <div class="signup-form-buttons">
+            <button class="signup-form-button" :class="{'active-theme' :activePath}">Register</button>
```

Comment:
this is super reduntant, could you make one SignUp component which sould support the login and register modes? This is why it was designed this way, to limit the unecessary copying